streams/src/main/java/org/apache/kafka/streams/kstream/KTable.java

```diff
@@ -316,4 +316,10 @@
      * @param action an action to perform on each element
      */
     void foreach(ForeachAction<K, V> action);
+
+    /**
+     * Get the state store name, if any
```

Comment:
nit: Get the name of the local state store used for materializing this {@link KTable}.